roles/dev/meta/main.yml

```diff
@@ -17,7 +17,6 @@ dependencies:
 - { role: etcd }
 - { role: docker }
 - { role: swarm }
-- { role: ucp }
```

Comment:
Just inline with my comment around the purpose of `dev` role, `ucp` can continue to stay here as when `dev` role is invoked with `prebake-for-dev` it automatically filters out all the tasks that are not needed in build environment.